aries_cloudagent/protocols/connections/v1_0/manager.py

```diff
@@ -482,6 +482,8 @@ async def create_response(
             if default_endpoint:
                 my_endpoints.append(default_endpoint)
             my_endpoints.extend(self.context.settings.get("additional_endpoints", []))
+        else:
```

Comment:
`if not` ... `else` is a sign of overthinking, could you reverse this and then I'll merge it?